_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,126 @@
+
+In this blog, you'll learn to create a new detector or monitor and add monitors and detectors to a visualization by way of the visualization, instead of through the Alerting or Anomaly Detection plugin pages.
+
+## Terms to know
+
+Here are terms you need to know:
+
+- _OpenSearch Dashboards visualizations_ are visualizations defined within a dashboard.
+- _Anomaly Detection plugin_ is the free OpenSearch plugin using the [Random Cut Forest algorithm](https://github.com/aws/random-cut-forest-by-aws) (that is, an unsupervised algorithm for detecting anomalous data points within a dataset) to detect anomalies in aggregated data.
+- _Alerting plugin_ is the free OpenSearch plugin used to monitor and trigger alerts on OpenSearch's health and machine-generated logs.
+
+## Getting started
+
+The alerting and anomaly detection visualizations feature is automatically enabled. You can turn it off by going to **OpenSearch Dashboards** > **Stack Management** > **Advanced Settings** and toggling off **Enable plugin augmentation**, which is in the **Visualization** pane. Before getting started with this feature, make sure you have:
+
+* Installed OpenSearch and OpenSearch Dashboards version 2.9 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
+* Installed Anomaly Detector or Alerting plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) for more information.
+* Started your local environment. Because OpenSearch Playground is read-only, use your local environment to perform the steps in the following tutorials.
+
+Note that currently only y-axis metric (count, average, max, etc.) aggregations and x-axis date histogram aggregations are supported. 
+
+### Creating a monitor or detector through a visualization
+
+To create a new monitor or detector by way of a visualization, first [save your dashboard]({{site.url}}{{site.baseurl}}/dashboards/dashboard/index/#saving-dashboards). Then select **Alerting** or **Anomaly Detection** from the **Options** context menu dropdown. The following image gives you a snapshot of the interface. Tip: If you are using static thresholds (for example, metrics to monitor CPU spikes, memory usage, or disk usage), create an alerting monitor; otherwise, create an anomaly detector.
+
+<img width="720" alt="dashboard-options-context-menu" src="https://github.com/vagimeli/project-website/assets/105296784/bee46a40-f674-482c-bd21-b288981f1d52">
```

Comment:
nit: I imagine we want the assets to be in the core repository instead of your forked one.

Reply:
Yes, I've added them to the core repo and replaced the images herein.